src/Agent.Worker/Handlers/Handler.cs

```diff
@@ -26,6 +26,10 @@ public interface IHandler : IAgentService
 
     public abstract class Handler : AgentService
     {
+        // The maximum supported size of a environment variable value is 32k, value lengthier than this isn't stored
```

Comment:
Also add something about: When launch process in windows, you can set environment variable greater then 32K, but that variable will not be able to read in node.exe.

Reply:
Done.